.github/workflows/mypy_primer.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
             --type-checker knot \
             --old base_commit \
             --new "$GITHUB_SHA" \
-            --project-selector '/(mypy_primer|black|pyp|git-revise|zipp|arrow|isort|itsdangerous|rich|packaging|pybind11|pyinstrument|typeshed-stats|scrapy|werkzeug|bidict|async-utils)$' \
+            --project-selector '/(mypy_primer|black|pyp|git-revise|zipp|arrow|isort|itsdangerous|rich|packaging|pybind11|pyinstrument|typeshed-stats|scrapy|werkzeug|bidict|async-utils|python-chess|dacite|python-htmlgen|paroxython|porcupine|psycopg)$' \
```

Comment:
I added a few projects with dataclass usage, some of which use `order=True`, but still no ecosystem changes. The usage of the order-methods is often hidden, e.g. by adding dataclass instances to a list which will then be sorted. We did not previously emit false positives in these cases (would rely on generics), so there are obviously no changes now.